events/app.py

```diff
@@ -39,28 +40,23 @@ def __getattribute__(self, _):
 
 @app.route('/v1/add', methods=['POST'])
 def add_event():
-    # TODO(cmei4444): Get event information from request data
-    info = request.form['info']
-    # TODO(cmei4444): Verify that user has event editing access
-    current_time = datetime.datetime.now()
-    info = build_event_info(info, current_time)
     try:
+        info = {
+            'name': request.form['event_name'],
+            'description': request.form['description'],
+            'author': request.form['author_id'],
```

Comment:
Is this input from the user? If so, we should use the id from the auth token instead.

Reply:
I was planning on retrieving the auth token ID in pageserve and posting it over with the rest of the information - not sure if this is the most secure way but I'm currently working on implementing it

Reply:
Agreed. That goes with the TODO on line 50. We'll need to get the gauth token from the request, make a request to the users service for authentication, check for authorization, then add the ID to info. Maybe that's better left to a separate PR.

Reply:
It's probably best to do that in another PR, clarified the TODO in 0e29b99

Reply:
Sounds good.